src/lib/y2network/connection/connection.rb

```diff
@@ -0,0 +1,20 @@
+require "y2network/interface"
+
+module Y2Network
+  module Connection
+    class Connection
```

Comment:
i don't understand the name. What the "Connection" means here? I'd expect e.g. PTP connection but not a class carrying interface configuration.

Reply:
We could change the name to "Config" or whatever (we used the network manager term). This class basically holds the configuration (and for the time being is related to an interface).

Reply:
It is based somehow on the idea of permiting having different connection profiles. That could belongs to an specific interface or not, wicked also support nanny policies that would be nice to support in the future. So we are trying to create a model that permits us to handle that.

Reply:
I understand what it tries to do. Just wanted to point that "Connection" is a bit misleading term in networking. It could be more precise, like "ProfileConnection" or something even better.

Reply:
Used "ConnectionConfig" namespace